mycore-ocfl/src/main/java/org/mycore/ocfl/metadata/migration/MCROCFLMigration.java

```diff
@@ -150,20 +187,30 @@ private void migrateID(String id) {
         }
     }
 
-    private MigrationStep migrateRevision(MCRAbstractMetadataVersion rev, MCRObjectID objectID) throws IOException {
+    private void writeRevision(MCROCFLRevision step, MCRObjectID objectID) throws IOException {
+        switch (step.type()) {
```

Comment:
What about the warning " Switch statements should be exaustive - add a default...."?